documentation/functions.asciidoc

```diff
@@ -379,6 +385,9 @@ The version is saved as `«tool»_VERSION` variable in `settings/devon.propertie
 Two version numbers are passed to the doVersionCompare function as parameters.
 If the versions are equal, the function returns 0, if the first version is higher than the second, returns 1, and if the second version is higher than the first, the function returns 2.
 
+=== doVersionWarning
```

Comment:
nice to have, but `doReportVersionSecurityWarning` would be a more explicit and self-explanatory name for the function.

Reply:
Of course. That was the name I was looking for